src/content/roadmap/statelessness/index.md

```diff
@@ -0,0 +1,98 @@
+
+History expiry is somewhat controversial because so far Ethereum has always implicitly guaranteed the availability of any historical data. A full sync from genesis has always been possible as standard, even if it relies on rebuilding some older data from snapshots. History expiry moves the responsibility for providing this guarantee outside of the Ethereum core protocol. This could introduce new censorship risks if it is centralized organizations that end up stepping in to provide historical data.
+
+EIP-4444 is not yet ready to ship, but it is under active discussion. Interestingly, the challenges with EIP-4444 are not so much technical, but mostly community management. In order for this to ship, there needs to be community buy-in that inludes not only agreement but also commitments to store and serve historical data from trustworthy entities.
+
+This upgrade doesn't fundamentally change how Ethereum nodes handle state data, it just changes how historical data is accessed.
+
+Watch [Alex Stokes explain EIP-4444](https://youtu.be/SfDC_qUZaos)
+Read the [EIP-4444 specification](https://eips.ethereum.org/EIPS/eip-4444)
+
+### State expiry {#state-expiry}
+
+State expiry refers to removing state from individual nodes if it hasn't been accessed recently. There are several ways this could be implemented, including:
+
+- **Expire by rent**: charging "rent" to accounts and expiring them when their rent reaches zero
+- **Expire by time**: making accounts inactive if there are no reading/writing to that account for some amount of time
+
+Expiry by rent could be a direct rent charged to accounts to keep them in the active state database. Expiry by time could be by countdown from the last account interaction, or it could be periodic expiry of all accounts. There could also be mechanisms that combine elements of both the time and rent based-models, for example individual accounts persist in the active state if they pay some small fee prior to time based expiry. With state expiry it is important to note that inactive state is **not deleted**, it is just stored separately from the active state. The inactive state can be resurrected into the active state.
+
+The way this would work is probably to have a state tree for specific time periods (perhaps ~1 year). Whenever a new period begins, so does a completely fresh state tree. Only the current state tree can be modified, all others are immutable. Ethereum nodes are only expected to hold the current state tree and the next most recent one. This requires a way to time-stamp an address with the period it exists in. This requires [addresses to be lengthened](https://ethereum-magicians.org/t/increasing-address-size-from-20-to-32-bytes/5485) to accommodate the additional information with the added benefit that longer addresses are much more secure. The roadmap item that does this is called [address space extension](https://ethereum-magicians.org/t/increasing-address-size-from-20-to-32-bytes/5485).
```

Comment:
There are alternatives to longer addresses, but ASE is definitely the preferred approach: https://ethereum-magicians.org/t/types-of-resurrection-metadata-in-state-expiry/6607